src/Sarif/Baseline/ResultMatching/DataStructures/MatchedResults.cs

```diff
@@ -72,9 +72,9 @@ public Result CalculateNewBaselineResult()
                 OriginalResultMatchingProperties = new Dictionary<string, object>();
             }
 
-            if (PreviousResult.OriginalRun.InstanceGuid != null)
+            if (PreviousResult.OriginalRun.Id.InstanceGuid != null)
```

Comment:
>. [](start = 45, length = 1)

"." => "?."

Reply:
Most of the changed lines below contain multi-level property accesses that require "?." at each level after the first.

---
In reply to: [226746235](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226746235) [](ancestors = 226746235)

Reply:
Good catch. This is a problem that this breaking change is likely to provoke for others as well.

---
In reply to: [226746466](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226746466) [](ancestors = 226746466,226746235)

Reply:
You missed Line 102.

---
In reply to: [226844027](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226844027) [](ancestors = 226844027,226746466,226746235)

Reply:
Resolved in subsequent PR.

---
In reply to: [226873134](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226873134) [](ancestors = 226873134,226844027,226746466,226746235)